### DISCUSSION TITLE
Unable to use Phone(SMS) authentication

### DISCUSSION THREAD

**Discussion by indersen** [supabase/supabase#15186] [Questions]
I am using WeWeb as frontend. I want to authenticate users using Phone SMS. I have setup Twilio for this. BUT, the plugin [supabase-auth](https://github.com/weweb-assets/plugin-supabase-auth) doesn't seem to have this functionality. 

Question; How can I achieve SMS Authentication with my WeWeb + Supabase setup?

**Comment by lukas-9888** [comment]
I am facing the same issue, let me know if you find an answer
